src/odemis/util/raster.py

```diff
@@ -142,3 +142,45 @@ def point_in_polygon(p, polygon):
         p1x, p1y = p2x, p2y
 
     return inside
+
+
+def get_possible_intersections(row_pairs: List[Tuple], col_pairs: List[Tuple], include_neighbors: bool = True) -> Set[Tuple]:
+    """
+    Uses Bresenham's line algorithm to determine the possible intersected fields/tiles.
+
+    :param row_pairs: (list of tuple) A list of (start_row, end_row) pairs defining line segments.
+    :param col_pairs: (list of tuple) A list of (start_col, end_col) pairs defining line segments.
+    :param include_neighbors: (bool) If True, includes adjacent neighbors to ensure complete coverage. Defaults to True.
```

Comment:
In the docstring, not need to indicate the type, if it's already fully defined in the type annotation, which is the case here.
Similarly, no need to indicate the default value, if it's clear by reading the method signature, which is the case here.